pages/get-involved.tsx

```diff
@@ -129,6 +130,19 @@ function Partner() {
   );
 }
 
+function GetInolvedFAQ() {
```

Comment:
Can you please correct the spelling, missing a "v" 😄

Reply:
@e-papanicolas lmaoo sorry about that. clearly my illiteracy has reached crippling levels